nixops/diff.py

```diff
@@ -31,12 +30,10 @@ def _default_handle(self):
         """
         raise NotImplementedError
 
-    def get_deps(self):
-        # type: () -> List[Handler]
+    def get_deps(self) -> List[Handler]:
         return self._dependencies
 
     def get_keys(self, *_: AnyStr) -> List[str]:
```

Comment:
I don't even know what this code is doing :o

Reply:
To be honest, me neither. :)

Reply:
A handler is basically a handler of few attributes e.g `handler(['region', 'instanceType'])`, which means it realizes the needed changes when those attributes are changed,
I think get_keys is just convenience to return the keys, don't remember why I added it to be honest :D

Reply:
or are you guys asking about `*_: AnyStr` which I don't what it does either :-)